DQM/SiPixelPhase1Config/python/SiPixelPhase1OnlineDQM_Timing_cff.py

```diff
@@ -0,0 +1,183 @@
+      Specification(OverlayCurvesForTiming).groupBy("DetId/Event") # per-layer with history for online
+                               .reduce("COUNT")
+                               .groupBy("PXForward/PXDisk/OnlineBlock") 
+                               .groupBy("PXForward/PXDisk", "EXTEND_Y")
+                               .save()
+  )
+
+#StandardSpecifications1D_Num.append(
+#      Specification(OverlayCurvesForTiming).groupBy("PXBarrel/OnlineBlock/PXLayer/Event") # per-layer with history for online
+#                     .reduce("COUNT")
+#                     .groupBy("PXBarrel/OnlineBlock") 
+#                     .groupBy("PXBarrel", "EXTEND_Y")
+#                     .save()
+#  )
+
+
+StandardSpecifications1D_Num.append(
+Specification(OverlayCurvesForTiming).groupBy("DetId/Event")
+                     .reduce("COUNT")
+                     .groupBy("PXBarrel/OnlineBlock")
+                     .groupBy("PXBarrel", "EXTEND_Y")
+                     .save()
+)
+
+
+StandardSpecifications1D_Num.append(
+Specification(OverlayCurvesForTiming).groupBy("DetId/Event")
+                     .reduce("COUNT")
+                     .groupBy("PXForward/OnlineBlock")
+                     .groupBy("PXForward", "EXTEND_Y")
+                     .save()
+)
+
+#StandardSpecifications1D_Num.append(
+#      Specification(OverlayCurvesForTiming).groupBy("PXForward/OnlineBlock/PXDisk/Event") # per-layer with history for online
+#                     .reduce("COUNT")
+#                     .groupBy("PXForward/OnlineBlock") 
+#                     .groupBy("PXForward", "EXTEND_Y")
+#                     .save()
+#  )
+#
+  
+# To Configure Phase1 DQM for Phase0 data
+SiPixelPhase1Geometry.upgradePhase = 0
+SiPixelPhase1Geometry.onlineblock = 5
+# Turn on 'online' harvesting. This has to be set before other configs are 
+# loaded (due to how the DefaultHisto PSet is later cloned), therefore it is
+# here and not in the harvestng config.
+DefaultHisto.perLumiHarvesting = True
+DefaultHistoDigiCluster.perLumiHarvesting = True
+DefaultHistoSummary.perLumiHarvesting = True
+DefaultHistoTrack.perLumiHarvesting = True
+
+
+# Pixel Digi Monitoring
+from DQM.SiPixelPhase1Digis.SiPixelPhase1Digis_cfi import *
+SiPixelPhase1DigisAnalyzer.src = cms.InputTag("siPixelDigis") # adapt for real data
+
+# Cluster (track-independent) monitoring
+from DQM.SiPixelPhase1Clusters.SiPixelPhase1Clusters_cfi import *
+
+# Track cluster 
+from DQM.SiPixelPhase1TrackClusters.SiPixelPhase1TrackClusters_cfi import *
+from DQM.SiPixelPhase1TrackResiduals.SiPixelPhase1TrackResiduals_cfi import *
+
+
+# Raw data errors
+from DQM.SiPixelPhase1RawData.SiPixelPhase1RawData_cfi import *
+
+from DQM.SiPixelPhase1Common.SiPixelPhase1GeometryDebug_cfi import *
+
+from DQM.SiPixelPhase1TrackEfficiency.SiPixelPhase1TrackEfficiency_cfi import *
+#Summary maps
+#from DQM.SiPixelPhase1Summary.SiPixelPhase1Summary_cfi import *
+
+
+siPixelPhase1OnlineDQM_source = cms.Sequence(
```

Comment:
would be great if this implementation was shared with the offline cff... (its mostly a duplication)